ipu-plugin/pkg/ipuplugin/ovsbridge.go

```diff
@@ -34,32 +34,32 @@ func createDbParam(ovsDbPath string) string {
 }
 
 func isNumeric(s string) bool {
-    var n int
-    _, err := fmt.Sscanf(s, "%d", &n)
-    return err == nil
+	var n int
+	_, err := fmt.Sscanf(s, "%d", &n)
+	return err == nil
 }
 
 func getPIDsWithComm(target string) ([]string, error) {
-    files, err := os.ReadDir("/proc")
-    if err != nil {
-        return nil, fmt.Errorf("error reading /proc: %v", err)
-    }
-
-    var pids []string
-    for _, file := range files {
-        if file.IsDir() && isNumeric(file.Name()) {
-            cmdPath := filepath.Join("/proc", file.Name(), "cmdline")
-            data, err := os.ReadFile(cmdPath)
-            if err != nil {
-                continue
-            }
-            if strings.Contains(strings.TrimSpace(string(data)), target) {
-                pids = append(pids, file.Name())
-            }
-        }
-    }
-
-    return pids, nil
+	files, err := os.ReadDir("/proc")
```

Comment:
Just the indentation changes at this function right?.

Reply:
Yes. Go Lint is integrated with my editor and it converts all spaces to tabs for golang which is the consistent way throughout rest of our code. The easier way would be to ignore whitespaces on github PR review setting